platform/ios/Mapbox-iOS-SDK-snapshot-dynamic.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |m|
 
-  version = '5.4.0-alpha.2'
+  version = '5.3.1'
```

Comment:
We don’t want to downgrade the podspec versions, since `master` is well beyond `5.3.x` at this point.